CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.0.3
+Added ability to raise specify the response and response headers
```

Comment:
wording here looks like it would be 
"Added ability to specify the response and response headers"
?